apps/pie-storybook/stories/pie-checkbox-group.stories.ts

```diff
@@ -66,32 +73,30 @@ export default checkboxGroupStoryMeta;
 const Template = ({
     name,
     label,
+    isInline,
     assistiveText,
     status,
     disabled,
 }: CheckboxGroupProps) => html`
-    <p>Please note, the checkboxes are separate components. See
-    <pie-link href="/?path=/story/checkbox--default">pie-checkbox</pie-link>.</p>
-    <pie-checkbox-group
-        name="${ifDefined(name)}"
-        label="${ifDefined(label)}"
-        assistiveText="${ifDefined(assistiveText)}"
-        status=${ifDefined(status)}
-        ?disabled="${disabled}"
-    >
-        <pie-checkbox
-            name="checkbox-one"
-            label="checkbox 1">
-        </pie-checkbox>
-        <pie-checkbox
-            name="checkbox-two"
-            label="checkbox 2">
-        </pie-checkbox>
-        <pie-checkbox
-            name="checkbox-three"
-            label="checkbox 3">
-        </pie-checkbox>
-    </pie-checkbox-group>
+    <div style="max-width: 400px">
+        <p>Please note, the checkboxes are separate components. See
+        <pie-link href="/?path=/story/checkbox--default">pie-checkbox</pie-link>.</p>
+        <pie-checkbox-group
+            name="${ifDefined(name)}"
+            label="${ifDefined(label)}"
+            assistiveText="${ifDefined(assistiveText)}"
+            ?isInline=${isInline}
+            status=${ifDefined(status)}
+            ?disabled="${disabled}"
+        >
+            <pie-checkbox name="checkbox-one">checkbox 1</pie-checkbox>
+            <pie-checkbox name="checkbox-two">checkbox 2 longer label</pie-checkbox>
+            <pie-checkbox name="checkbox-three">checkbox 3</pie-checkbox>
+            <pie-checkbox name="checkbox-three">checkbox 4</pie-checkbox>
+            <pie-checkbox name="checkbox-three">checkbox 5 even longer label</pie-checkbox>
```

Comment:
Will labels ever wrap to another line?

Reply:
I updated the story to show a checkbox with a label that goes to a second line